docs/modules/ROOT/pages/vshn-managed/how-tos/custom-image-keycloak.adoc

```diff
@@ -0,0 +1,30 @@
+= How to create a custom Docker image for Keycloak
```

Comment:
To me, this sounds like creating a custom image for Keycloak itself. This needs a better title and an introduction to what it's all about.

Reply:
Check it out now. Now it should be clear.

Reply:
Much better, thanks